### ISSUE TITLE
Blinking Thumbnail

### ISSUE THREAD

**Issue by soaibsafi** [bep/gallerydeluxe#28]
While scrolling, the thumbnails are blinking on the iPhone. 
Tested on Safari and Chrome.
There seems to be a delay in changing the position of the thumbnails.

![RPReplay_Final1716710958-ezgif com-resize](https://github.com/bep/gallerydeluxe/assets/11424812/3266874c-19a5-4830-b205-09d1f9177f11)

**Comment by bep** [comment]
Yes, I have noticed this myself. I will put this on my "top priority" list.

**Comment by bep** [comment]
OK, so testing it now on my iPhone SE I cannot reproduce this "flicker" thing. 

What iPhone do you run and is IOS/Chrome/Safari up to date?

**Comment by soaibsafi** [comment]
I tested it on iPhone 12 
iOS/Safari Version: 17.5.1
Chrome Version: 125.0.6422.80

**Comment by bep** [comment]
@soaibsafi OK, maybe I was seeing this on my 2nd gen iPhone SE (I have that lying around somewhere). I'm not on iPhone SE 3rd gen.

**Comment by midzer** [comment]
No flicker at all while scrolling with Safari on iOS 17.5.1 using a iPhone SE 3.

@soaibsafi Do you have any browser extensions running?

**Comment by soaibsafi** [comment]
@midzer I have Grammarly installed, but it's not running. 

I don't know if the issue is with the iPhone 12. I also asked a friend to test it on his iPhone 14, but the issue doesn't exist there.
